docs/src/content/hardhat-chai-matchers/docs/migrate-from-waffle.md

```diff
@@ -90,38 +90,8 @@ The `@nomicfoundation/hardhat-chai-matchers` plugin is meant to be a drop-in rep
 
    ::::
 
-4. If you were not importing the `@nomiclabs/hardhat-ethers` plugin explicitly (because the Hardhat Waffle plugin already imported it), then add it to your config:
-
-   ::::tabsgroup{options=TypeScript,JavaScript}
-
-   :::tab{value=TypeScript}
-
-   ```ts
-   import "@nomiclabs/hardhat-ethers";
-   ```
-
-   :::
-
-   :::tab{value=JavaScript}
-
-   ```js
-   require("@nomiclabs/hardhat-ethers");
-   ```
-
-   :::
-
-   ::::
-
 :::tip
 
 Looking for a replacement of Waffle's `loadFixture`? You can find our version of it in [Hardhat Network Helpers](/hardhat-network-helpers/docs/reference#fixtures).
 
 :::
-
-## Why migrate?
```

Comment:
Happy to see this section go, tbh. It's all well covered in the Overview, so good to de-dupe.